netbox_kea/templates/netbox_kea/inc/reservation_sync_badge.html

```
@@ -0,0 +1,15 @@
+{% if state.label == "Synchronized" %}
```

HTMLHint (1.9.2)

[error] Doctype must be declared before any non-comment content.

(doctype-first)